src/FMData.Rest/FileMakerRestClient.cs

```diff
@@ -61,9 +62,10 @@ public class FileMakerRestClient : FileMakerApiClientBase, IFileMakerRestClient
         /// <param name="file">Name of the FileMaker Database to connect to.</param>
         /// <param name="user">Account to connect with.</param>
         /// <param name="pass">Account to connect with.</param>
+        /// <param name="targetVersion">Version of the DataAPI endpoint to use. Default is v1.</param>
         [Obsolete("Creates a new HttpClient for this instance, and is generally not good. Inject a managed client.")]
-        public FileMakerRestClient(string fmsUri, string file, string user, string pass)
-            : this(new HttpClient(), new ConnectionInfo { FmsUri = fmsUri, Database = file, Username = user, Password = pass }) { }
+        public FileMakerRestClient(string fmsUri, string file, string user, string pass, RestTargetVersion targetVersion = RestTargetVersion.v1)
+            : this(new HttpClient(), new ConnectionInfo { FmsUri = fmsUri, Database = file, Username = user, Password = pass, RestTargetVersion = targetVersion }) { }
```

Comment:
Don't want to be making changes to Obsolete'd constructors here and I think its safe to revert this, with the Target Version property living on the ConnectionInfo class the other overloads should handle this.